SPEC.md

```diff
@@ -1574,14 +1604,17 @@ Example: string_to_file.wdl
 version 1.2
 
 workflow string_to_file {
-  String path1 = "/path/to/file"
-  File path2 = "/path/to/file"
+  input {
+    File infile
+  }
+
+  String path1 = "~{infile}"
 
   # valid - String coerces unambiguously to File
-  File path3 = path1
+  File path2 = path1
 
   output {
-    Boolean paths_equal = path2 == path3
+    Boolean paths_equal = path1 == path2
```

Comment:
You can still check the exact same thing as the original test by making `infile` be `path1`, `path1` be `path2` and `path2` be `path3`. Then you assert `path2 == path3` in the output.